### DISCUSSION TITLE
Como saber se estou pronto para começar a fazer freelas?!?

### DISCUSSION THREAD

**Discussion by igorpinhh** [frontendbr/forum#1213] [Discussão]
Olá, há certo tempo decidi sair da zona de conforto e começar a pesquisar lugares diferentes pra trabalhar. Porém, eis que me surgiu a dúvida esta dúvida e gostaria de saber de vocês que são mais experientes, como começaram a carreira de freelancer? Como descobriram que estavam preparados pra começar? 
Obs: eu não tenho experiência nenhuma (profissionalmente) e sou do interior.

**Comment by kavalcante** [comment]
Olha, eu não trabalho com freelas (100% como freelancer), porém já fiz, provavelmente como todo dev. Acho que a única resposta que tenho pra isso é, eu comecei a procurar freelas, dai tu pode utilizar plataformas ou indicações em grupos e afins. Lia a descrição, entendia o que a pessoa queria. Se eu conseguisse fazer aquilo, eu pegava o freela. Se não, não.

As vezes a descrição não é suficiente para entender, daí você pode conversar com a pessoa e entender se tu consegue ajudar.

Acho que o importante é ser honesto também. Saber suas limitações e ir estudando e fazendo as coisas. Isso não significa que você precisa se jogar para baixo. Porém, em um freela, é legal que você saiba o que você está fazendo, ou pelo menos, saiba como chegar no objetivo final, mesmo que isso signifique que você tenha que estudar alguma coisa. Porém, cuidado com os prazos. Se você não domina muito uma tecnologia e quer pegar um freela dela, coloque prazos mais largos.

**Comment by expalmer** [comment]
RE: Començando!

**Comment by bernardodiasc** [comment]
> Como começaram a carreira de freelancer?

Acha alguem mais proximo que tenha uma demanda e oferece o serviço. Sendo alguem mais proximo voce pode ser transparente com a sua experiencia e intenção. Na minha experiência algumas pessoas foram verdadeiros anjos no começo da minha carreira. Mas precisou ousadia da minha parte e um bocado de papo de vendedor.

> Como descobriram que estavam preparados pra começar?

Se vc quer, vc ta preparado pra começar. :)

**Comment by MichelleTrigueiro** [comment]
> 
> > Como descobriram que estavam preparados pra começar?
> 
> Se vc quer, vc ta preparado pra começar. :)

**Achei muito bakana sua resposta: "Se você quer, vc ta preparado para começar". Muito obrigada e traduz bastante meu atual sentimento.**

**Comment by thismarcoantonio** [comment]
Essa issue me deixou com uma pulga atras da orelha, 

> Se vc quer, vc ta preparado pra começar. :)

Eu estava pensando em pegar freelas, mas esse tipo de frase me motiva e me dá vontade de correr atrás. Porém, mesmo com experiência na área, sinto dificuldades em encontrar clientes ou projetos para desenvolver. Se alguém tiver dicas ou cases de como começaram e quais passos seguiram para conseguir freelas constantes.

**Comment by praganavictor** [comment]
Acredito que tenham varios artigos sobre isso na internet mas aqui está oque eu mais gostei sobre como se tornar um freelancer

https://onebitcode.com/programador-freelance-remote/

**Comment by bernardodiasc** [comment]
> Eu estava pensando em pegar freelas, mas esse tipo de frase me motiva e me dá vontade de correr atrás. Porém, mesmo com experiência na área, sinto dificuldades em encontrar clientes ou projetos para desenvolver. Se alguém tiver dicas ou cases de como começaram e quais passos seguiram para conseguir freelas constantes.

Tem muito material online, mas a sua intuição e aprendizado práticos são o que contam mesmo. A maior sacada é que voce tem que se dar conta de que vai precisar atuar em diversas frentes diferentes, vai muito além do produto/serviço que voce está vendendo. Isso é importante pois vai te permitir inclusive formatar o que exatamente está vendendo. A expectativa do cliente deve ser sempre alcançada, mas voce define o que esta vendendo, vc define qual a expectativa.

Veja bem, essas experiências de trabalho autônomo são únicas, você pode até procurar inspiração e conselhos por ai afora, a teoria pode te dar bagagem MAS a experiência se adquire com experiência.

O que posso dar de dica é procurar entender como o seu produto/serviço encaixa no mercado, como a demanda dele surge, o que soluciona, quem é e onde está seu publico-alvo, etc. A única forma de descobrir essas informações é saindo na rua e vendendo, na primeira (tentativa de) venda tudo que vc sabe a respeito do que esta fazendo se desdobra num universo maior e vai desdobrando mais a medida que voce refina sua organização (organização no sentido de empresa mesmo, só que uma empresa de uma pessoa só). Isso é só a parte de venda, a parte comercial que conversa direto com quem paga o que quer que seja. Depois vc troca de chapéus, vai pro financeiro, volta pra produção, faz pós-venda e suporte, tambem faxina na cozinha e café pras visitas... hahaha
